src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
 // write your code here
+
+function formatCells(partOfTable) {
+  const rows = [...document.querySelector(partOfTable).rows];
```

Comment:
Ensure that the `partOfTable` parameter passed to `document.querySelector` is a valid selector and that the table structure supports this operation. If `partOfTable` does not match any element, `document.querySelector` will return `null`, leading to an error when trying to access `.rows`.